independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/spi/RuntimeConfiguration.java

```diff
@@ -12,6 +13,10 @@
 
     Limits limits();
 
+    boolean enableCompression();
+
+    Set<String> compressMediaTypes();
```

Comment:
shouldn't it be `compressed`?

Reply:
Well, we have `HttpConfiguration.compressMediaTypes` so I just wanted to keep this aligned. `HttpConfiguration.compressedMediaTypes` is ok though. Should I rename both?

Reply:
FTR I used the verb (`compress`) to keep consistency with `enableCompression`..